plugins/markdown/plugin-markdown-hint/src/client/styles/hint.scss

```diff
@@ -0,0 +1,284 @@
+@use 'sass:color';
+@use 'icons';
+@use 'svg';
+
+.hint-container {
+  position: relative;
+  transition:
+    background var(--vp-ct),
+    border-color var(--vp-ct),
+    color var(--vp-ct);
```

Comment:
I couldn’t find the definition of `var(--vp-ct)`.

Reply:
This.should.be.unified to --vp-tc with guideline, but the main branch is not including it now.